docs/intro/03-how-it-works.md

```diff
@@ -14,7 +14,28 @@ The server collects the results from all of the captured browsers and presents t
 
 This is only a very brief overview, as the internals of how Karma works aren't entirely necessary when using Karma.
 
-However, if you are interested in learning more, Karma itself originates from a university thesis, which goes into detail about the design
+## Outline of workflow.
+
+Here is roughly how Karma works:
+
+After starting up, Karma loads plugings and the configuration file, then starts its local web server which listened for connections.
+Any browser already watching the web address (localhost:9876 by default) will reconnect immediately. Karma also starts reporters
```

Comment:
Can you clarify that "already watching" means (I assume) already have that page loaded from an earlier run?

Reply:
Actually I think this is base on the websocket from the client page reconnecting. Changed the text.